packages/core/src/Modules/Cashier/Containers/account-transfer.jsx

```diff
@@ -40,10 +42,20 @@ const AccountTransfer = ({
         };
     }, []);
 
+    const hideNotes = () => {
+        if (typeof setSideNotes === 'function') {
+            setSideNotes(null);
+        }
+    };
+
+    BinarySocket.wait('authorize', 'website_status', 'get_settings', 'paymentagent_list').then(() => {
+        setIsLoadingStatus(false);
+    });
+
     if (is_virtual) {
         return <Virtual />;
     }
-    if (is_loading || is_switching) {
+    if (is_loading || is_switching || is_loading_status) {
```

Comment:
Why do we need two loading statuses?

Reply:
Yes, you’re right.

Reply:
We can use 'is_loading_status' because it’s covering 'is_loading' as well.

Reply:
Actually 'is_loading' is a state from cashier store and it can be changed in different situations. But 'is_loading_status' is for initial loading only.
I'll move 'BinarySocket.wait' function in to the init effect function.